modules/getNodePath.js

```diff
@@ -25,15 +27,44 @@ module.exports = function(type) {
 
     // global override?
     let globallySetType = Settings[`${type}Path`];
-    
+
     if (globallySetType) {
         resolvedPaths[type] = globallySetType;
     } else {
         var binPath = (Settings.inProductionMode)
             ? binaryPath.replace('nodes','node') + '/'+ type +'/'+ type
             : binaryPath + '/'+ type +'/'+ process.platform +'-'+ process.arch + '/'+ type;
 
-        if(Settings.inProductionMode) {
+        if(/*Settings.inProductionMode*/ true) {    
+            if(process.platform === 'linux' || 'darwin') {
+                var systemGethPath;
+                var proc = exec('which geth', 
+                    function(error, stdout, stderr) {
+                        systemGethPath = stdout.toString().replace(/\n/, '');
+                        if (error !== null) {
+                                log.error("error exec 'which geth': " + error + stdout);
+                        }
+                    });
+                
+                var systemGethVersion;
+                setTimeout(function () {
```

Comment:
I would recommend against using `which`. The `type` command is probably what we need, see http://superuser.com/questions/351889/what-is-the-unix-command-to-find-out-what-executable-file-corresponds-to-a-given